### PR TITLE
adding skill & location to assign hist TM-2768

### PR DESCRIPTION
**Pull request by SidelineCory24**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2416)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/343)

### CODE REVIEW

File: talentmap_api/fsbid/services/assignment_history.py [hunk 1]
```diff
@@ -56,24 +56,24 @@ def assignment_history_to_client_format(data):
                     "asgd_revision_num": x['asgd_revision_num'],
                     "position": {
                         "grade": pos.get("posgradecode", None),
-                        "skill": f"{pos.get('pos_skill_desc', None)} ({pos.get('pos_skill_code')})",
-                        "skill_code": pos.get("pos_skill_code", None),
+                        "skill": f"{pos.get('posskilldesc', None)} ({pos.get('posskillcode')})",
+                        "skill_code": pos.get("posskillcode", None),
```
[the remark this answers is not in the document]
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/assignment_history.py:60]: I spoke with Scott it is not clear if the skill2 is coming through on the WS side. Should I remove it on the mock instead, or finish setting it up here so it's ready to go once it is added?

[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:60]: oh yea, if it isn't coming through on the WS side then we don't want it in the mock

File: talentmap_api/fsbid/services/assignment_history.py [hunk 2]
```diff
@@ -56,24 +56,24 @@ def assignment_history_to_client_format(data):
...
                         "title": pos.get("postitledesc", None),
                         "post": {
-                            "code": loc.get("gvt_geoloc_cd", None),
-                            "post_overview_url": get_post_overview_url(loc.get("gvt_geoloc_cd", None)),
-                            "post_bidding_considerations_url": get_post_bidding_considerations_url(loc.get("gvt_geoloc_cd", None)),
-                            "obc_id": get_obc_id(loc.get("gvt_geoloc_cd", None)),
+                            "code": loc.get("locgvtgeoloccd", None),
+                            "post_overview_url": get_post_overview_url(loc.get("locgvtgeoloccd", None)),
+                            "post_bidding_considerations_url": get_post_bidding_considerations_url(loc.get("locgvtgeoloccd", None)),
+                            "obc_id": get_obc_id(loc.get("locgvtgeoloccd", None)),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/assignment_history.py:71]: let's either do `loc.get("locgvtgeoloccd")` or `or None`

File: talentmap_api/fsbid/services/assignment_history.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/assignment_history.py:60]: removed on mock side

File: talentmap_api/fsbid/services/assignment_history.py (hunk 2 above)
Comment (SidelineCory24) [reply talentmap_api/fsbid/services/assignment_history.py:71]: done

File: talentmap_api/fsbid/services/assignment_history.py [hunk 3]
```diff
@@ -56,24 +56,24 @@ def assignment_history_to_client_format(data):
...
                         "position_number": pos.get('posnumtext', None),
                         "position_id": x['position_id'],
                         "title": pos.get("postitledesc", None),
                         "post": {
-                            "code": loc.get("gvt_geoloc_cd", None),
-                            "post_overview_url": get_post_overview_url(loc.get("gvt_geoloc_cd", None)),
-                            "post_bidding_considerations_url": get_post_bidding_considerations_url(loc.get("gvt_geoloc_cd", None)),
-                            "obc_id": get_obc_id(loc.get("gvt_geoloc_cd", None)),
+                            "code": loc.get("locgvtgeoloccd") or None,
+                            "post_overview_url": get_post_overview_url(loc.get("locgvtgeoloccd") or None),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/assignment_history.py:69]: or None doesn't matter for this line, cause it's a () call instead of an assignment

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:69]: applies to both lines

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:69]: three lines lol

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:69]: sorry, i know i left this comment right next to those lines https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/759#discussion_r1117285433 but i just meant it as a general for all the changes

Comment (SidelineCory24) [reply talentmap_api/fsbid/services/assignment_history.py:69]: updated lol

Comment (SidelineCory24) [reply talentmap_api/fsbid/services/assignment_history.py]: ah okay, updating the rest

Comment (SidelineCory24) [reply talentmap_api/fsbid/services/assignment_history.py]: one line, both lines, three lines, ... 11 lines lol

File: talentmap_api/fsbid/services/assignment_history.py [hunk 4]
```diff
@@ -55,25 +55,25 @@ def assignment_history_to_client_format(data):
                     "asgd_tod_desc_text": x['asgd_tod_desc_text'],
                     "asgd_revision_num": x['asgd_revision_num'],
                     "position": {
-                        "grade": pos.get("posgradecode", None),
-                        "skill": f"{pos.get('pos_skill_desc', None)} ({pos.get('pos_skill_code')})",
-                        "skill_code": pos.get("pos_skill_code", None),
-                        "bureau": f"({pos.get('pos_bureau_short_desc', None)}) {pos.get('pos_bureau_long_desc', None)}",
+                        "grade": pos.get("posgradecode") or None,
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/assignment_history.py:58]: im not seeing us mapping language anywhere?

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:58]: disregard!

**Review by elizabeth-jimenez — APPROVED** [review]
🤌

**Pull request merged**